BUILD_WSL.md

```diff
@@ -0,0 +1,31 @@
+# Compiling hashcat for Windows with Windows Subsystem for Linux.
+
+Tested on a Windows 10 (x64 build 18363), should also work to build hashcat for Windows on Linux.
```

Comment:
should we really mention a build version here ? I think it will look outdated very soon, within some months/years. I think mentioning windows 10 64-bit is enough

Reply:
sure will remove, I'd guess that this will stay true for multiple versions